src/default_config_params/mod.rs

```diff
@@ -0,0 +1,66 @@
+use std::path::PathBuf;
+pub fn get_default_wallet_path() -> PathBuf {
+    let home_var = std::env::var("HOME");
```

Comment:
Don't bother fetching from an ENV variable. Thats not needed at this stage. The user defined path should be given in the `WalletStorage::init()` API as an `Option`, which will simply have `unwrap_or_default()`.

The default value is `get_default_datadir().append("wallet")`.